pkg/fqdn/dnsproxy/proxy_test.go

```diff
@@ -1038,3 +1043,103 @@ func Benchmark_perEPAllow_setPortRulesForID(b *testing.B) {
 		}
 	}
 }
+
+func Benchmark_perEPAllow_setPortRulesForID_large(b *testing.B) {
+	b.Skip()
+
+	bb, err := ioutil.ReadFile("testdata/cnps-large.yaml")
+	if err != nil {
+		b.Fatal(err)
+	}
```

Comment:
For now this benchmark will be skipped until we have a proper fixture for CNPs checked in.